lib/editor/components/EditorInput.js

```diff
@@ -118,31 +118,26 @@ export default class EditorInput extends Component<Props> {
     }
   }
 
-  _renderRouteTypeOptions = (field: any) => {
-    const standardTypes = []
-    const extendedTypes = []
-    const routeTypes = ['Standard', 'Extended']
-    const fieldOptions = field.options
-    for (const option in fieldOptions) {
-      if (fieldOptions[option].group === 'Standard') {
-        standardTypes.push(fieldOptions[option])
-      } else {
-        extendedTypes.push(fieldOptions[option])
+  _renderRouteTypeOptions = (field: GtfsSpecField): React.Node => {
```

Comment:
I think the more accurate output type would be `Array<React.Node>` (non-blocking)

Reply:
```suggestion
  _renderRouteTypeOptions = (field: GtfsSpecField): Array<React.Node> => {
```